### PR TITLE
Translated into Japanese. Right now it's just index, but I'll do the …

### PR DESCRIPTION
**Pull request by Mr-IjIj**
…others soon.

### CODE REVIEW

File: docs/ja/index.rst [hunk 1]
```diff
@@ -0,0 +1,53 @@
+unauthenticated users are able to access it::
```
Comment (markstory) [inline docs/ja/index.rst:1]: There seems to be some content missing here.

**Comment by othercorey** [comment]
Let's wait for the full translation here. Copying english to other language files means we have unsupported english that might not be translated.

**Comment by Mr-IjIj** [comment]
Thank you for checking, it looks like I was using the wrong branch to push. I'm very sorry for the inconvenience...
We sent you a different pull request.

**Comment by othercorey** [comment]
Moved to https://github.com/cakephp/authentication/pull/408

**Pull request closed without merging**